src/yunohost/backup.py

```diff
@@ -1776,8 +1776,13 @@ def mount(self, restore_manager):
         tar.close()
 
         # Mount the tarball
-        ret = subprocess.call(['archivemount', '-o', 'readonly',
-                               self._archive_file, self.work_dir])
+        try:
+            ret = subprocess.call(['archivemount', '-o', 'readonly',
```

Comment:
if archivemount is not there, subprocess.call() does not simply return something != 0 ?